services/api-server/docker/boot.sh

```diff
@@ -27,12 +27,16 @@ fi
 
 # RUNNING application ----------------------------------------
 if [ "${SC_BOOT_MODE}" = "debug-ptvsd" ]; then
-  # NOTE: ptvsd is programmatically enabled inside of the service
-  # this way we can have reload in place as well
-  exec uvicorn simcore_service_api_server.main:the_app \
-    --reload \
-    --host 0.0.0.0 \
-    --reload-dir services/api-server/src/simcore_service_api_server
+  reload_dir_packages=$(find /devel/packages -maxdepth 3 -type d -path "*/src/*" ! -path "*.*" -exec echo '--reload-dir {} \' \;)
```

Comment:
so ``find /devel/packages -maxdepth 3 -type d -path "*/src/*" ! -path "*.*" -exec echo '--reload-dir {}' \;`` means

- find directories
- on max depth 3  --> does this reach every directory?
- that is under ``src``
- and whose name does not contain a dot "."
- and set them as ``reload-dir``:  --> why we need to add all directories and not only the ``/src`` one? I would assume from [the doc](https://www.uvicorn.org/settings/#development) that provided one directory, it should check all files in it recursively (i.e. no need to explicitly add subdirectories in it!)

Reply:
BTW I also think that the last ``\'`` is not necessary. It could workw ith
```
reload_dir_packages=$(find /devel/packages -maxdepth 3 -type d -path "*/src/*" ! -path "*.*" -exec echo '--reload-dir{}'  \; )
```

Reply:
The above command will help you produce the below list of folders to be monitored, which are actual python packages:

- '/devel/packages/dask-task-models-library/src/dask_task_models_library'
- '/devel/packages/models-library/src/models_library'
- '/devel/packages/postgres-database/src/simcore_postgres_database'
- '/devel/packages/pytest-simcore/src/pytest_simcore'
- '/devel/packages/service-integration/src/service_integration'
- '/devel/packages/service-library/src/servicelib'
- '/devel/packages/settings-library/src/settings_library'
- '/devel/packages/simcore-sdk/src/simcore_sdk'

Reply:
aha! now i see. I interpreted it wrong